ensime-editor.el

```diff
@@ -685,6 +688,13 @@ Decide what line to insert QUALIFIED-NAME."
       (goto-char (point-at-bol))
       (funcall (funcall insert-import-fn insertion-range starting-point qualified-name)))))
 
+(defun ensime-ask-user-to-select-entry (title entries)
+  "Prompts the user to select an entry of entries"
+  (if ensime-use-helm
+      (ensime-helm-select-entry entries title)
+      (popup-menu* entries :point (point))
+    ))
```

Comment:
I know we're inconsistent, but we're trying to use standard lisp conventions of no dangling brackets. Also, I believe the standard formatted form for `if` is

```elisp
(if test
    then
  else)
```